ratdb/PMTCHARGE.ratdb

```diff
@@ -128,18 +120,47 @@ charge_prob: [259.22322874793593, 269.3602291533824, 279.7736994345442, 290.4652
+}
+
+//E.Keener at Penn, updates to charge and probabilities based on darkbox data from Berkeley Jun to Dec 2023.
```

Comment:
Might be worth including this reference in the comment: https://arxiv.org/abs/2311.05080